Core/clim-core/gadgets/concrete.lisp

```diff
@@ -1574,6 +1574,20 @@ if INVOKE-CALLBACK is given."))
   (popup-list-box pane)
   (disarm-gadget pane))
 
+(defmethod handle-event ((pane generic-option-pane) (event pointer-scroll-event))
+  (when (list-pane-exclusive-p pane)
+    (let ((delta-y (pointer-event-delta-y event))
+          (items (list-pane-items pane)))
+      (if (slot-boundp pane 'value)
+          (let* ((old-position (position (gadget-value pane) items
+                                         :test (list-pane-test pane)
+                                         :key (list-pane-value-key pane)))
```

Comment:
a brief testing reveals a small defect. I've tried `option-test` from clim examples and I've landed in the debugger when I've scrolled the second box, that's because the value NIL is not of the expected type NUMBER (old-position is bound to NIL here).

Here's why this is happening:
- the option-test demo has a bug, because the second pane is defined as:
```
(option-pane-2 :option-pane
                         :value "Option 1"
                         :items '("Option 1" "Option 2" "Option 3" "Option 4" "Option 6" "Option 7")
                         :value-changed-callback (constantly nil))
```

since the default test is EQL, the current value is not the member of ITEMS. That's a bug in a demo, however I'd like for this code to not land the user in a debugger here.

I've also checked, and the value in a value-gadget is always bound (by default to NIL), so checking whether the slot is bound is not necessary. how about

```
(if-let ((old-position …))
  (scroll)
  (initiate)
```

?

also we could drop the test for whether delta-y is plusp in the second test and simply set to car items, that perhaps would be more intuitive?